sdk/ml/azure-ai-ml/azure/ai/ml/constants/_component.py

```diff
@@ -54,9 +54,9 @@ class ExternalDataType(object):
 
 
 class DataTransferBuiltinComponentUri(object):
-    IMPORT_DATABASE = "azureml://registries/azureml-preview/components/import_data_database/versions/0.0.1"
-    IMPORT_FILE_SYSTEM = "azureml://registries/azureml-preview/components/import_data_file_system/versions/0.0.1"
-    EXPORT_DATABASE = "azureml://registries/azureml-preview/components/export_data_database/versions/0.0.1"
+    IMPORT_DATABASE = "azureml://registries/azureml/components/import_data_database/versions/0.0.1"
```

Comment:
please resolve this comment and complete this pr if you have succefully verify it in several regions.

Reply:
It can submit and run in Int region with adftest compute, but run failed with serveless compute. 

But I think it's not related with built-in component (azureml or azureml-preview), will check with chengbin later